cfgov/cfgov/urls.py

```diff
@@ -213,6 +214,12 @@
     ],
         namespace='transcripts')),
 
+    # TODO: Remove prototype landing page routes when all organisms and molecules have been implemented elsewhere.
```

Comment:
Could we make these routes only work locally so that only real pages get deployed to flapjack, refresh, and beta?